packages/portal/src/app/shared/errors/not-found/not-found.component.scss

```diff
@@ -0,0 +1,89 @@
+h1 {
+  font-family: 'Bungee Shade', cursive;
```

Comment:
Where is this font coming from? This is not a standard one, and we only use Roboto and the system one in the app.
Could you change it to match the existing design?

Reply:
Okay, just removed the unwanted fonts @sinedied @anfibiacreativa